api/illumio/cloud/k8scluster/v1/resource_config.proto

```diff
@@ -94,6 +95,185 @@ message KubernetesResourceMutation {
   }
 }
 
+// Information identifying a Cilium flow.
+// https://pkg.go.dev/github.com/cilium/cilium/api/v1/flow#section-readme
+message CiliumFlow {
+  google.protobuf.Timestamp time = 1;
+  string node_name = 2;
+  TrafficDirection traffic_direction = 3;
+  Verdict verdict = 4;
+  IP layer3 = 5;
+  Layer4 layer4 = 6;
+  CiliumEventType event_type = 8;
+  optional Endpoint source_endpoint = 9;
+  optional Endpoint destination_endpoint = 10;
+  NetworkInterface interface = 11;
+  int32 proxy_port = 12;
+  repeated Policy egress_allowed_by = 13;
+  // The CiliumNetworkPolicies allowing the ingress of the flow.
+  repeated Policy ingress_allowed_by = 14;
+  // The CiliumNetworkPolicies denying the egress of the flow.
+  repeated Policy egress_denied_by = 15;
+  // The CiliumNetworkPolicies denying the ingress of the flow.
+  repeated Policy ingress_denied_by = 16;
+}
+
+// Information identifying the direction of traffic in Cilium.
+// https://pkg.go.dev/github.com/cilium/cilium/api/v1/flow#readme-trafficdirection
+enum TrafficDirection {
```

Comment:
Many of those messages seem to be unchanged compared to Cilium's, and that's good.
However, it's not optimal to just copy-and-paste all of them.
How could you refer to it directly? This will require playing with Buf, incl. maybe downloading and caching locally the .proto file from the Cilium repo, but that should be possible.